packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -15,7 +15,8 @@ import '../runner/flutter_command.dart';
 class ConfigCommand extends FlutterCommand {
   ConfigCommand({ bool verboseHelp = false }) {
     argParser.addFlag('analytics',
```

Comment:
do we want to hide this one by default, since it's redundant with the other ones?

Reply:
I think that's a good idea, i have hidden it so that it only shows up when running in verbose mode